### DISCUSSION TITLE
In what ways can we stand out in terms of the features offered by our Proxy

### DISCUSSION THREAD

**Discussion by sleipnir** [eigr/massa#36] [Ideas]
Explore ideas that set us apart from other implementations. I cast some points where we could start:

* Inter cluster Sideeffects and forwards
* External Sideeffects and forwards
* Supported libraries with Unix Domain Sockets
* Wasi support
* Tensor Entities (using Elixir Nx)

Anymore?

**Comment by ralphlaude** [comment]
I will go with all those topics even if I understand only the third first ones :)

**Reply by sleipnir** [reply]
If we manage to reach the top three it will be a great achievement.
If you have anything else that can be added to the list I would be happy to know.
